src/main/webapp/app/course/plagiarism-cases/instructor-view/detail-view/plagiarism-case-instructor-detail-view.component.ts

```diff
@@ -159,36 +183,30 @@ export class PlagiarismCaseInstructorDetailViewComponent implements OnInit, OnDe
-            this.plagiarismCase.exercise!.title
-        } in the course ${
-            this.plagiarismCase.exercise!.course!.title
-        }, we have concluded that you have committed plagiarism.\nThis is not only a violation of principles of good student practice but also of the “Student Code of Conduct” of the faculty of computer science that you have agreed upon. You can check the Student conduct code [here](https://www.in.tum.de/fileadmin/w00bws/in/2.Fur_Studierende/Pruefungen_und_Formalitaeten/1.Gute_studentische_Praxis/englisch/leitfaden-en_2016Jun22.pdf) §22.1 of the “Allgemeine Studien- und Prüfungsordnung (APSO)” [“General Examination and Study Regulations”] regulates consequences for such cases. You can find the APSO [here](https://www.tum.de/studium/im-studium/das-studium-organisieren/satzungen-ordnungen#statute;t:Allgemeine%20Prüfungs-%20und%20Studienordnung;sort:106;page:1).\nYou have one week to provide a statement about this situation.`;
-        this.createdPost.title = `Plagiarism Case ${this.plagiarismCase.exercise!.title}`;
-    }
+        const studentName = this.plagiarismCase.student!.name;
+        const courseTitle =
+            this.plagiarismCase.exercise!.course!.title!.length > 40
+                ? this.plagiarismCase.exercise!.course!.title?.slice(0, 35) + '…'
+                : this.plagiarismCase.exercise!.course!.title;
 
-    /**
-     * Get the url segment for different types of exercises.
-     * @param exerciseType type of exercise
-     */
-    getExerciseUrlSegment(exerciseType?: ExerciseType) {
-        switch (exerciseType) {
-            case ExerciseType.TEXT:
-                return 'text-exercises';
-            case ExerciseType.MODELING:
-                return 'modeling-exercises';
-            case ExerciseType.PROGRAMMING:
-                return 'programming-exercises';
-            default:
-                throw Error('Unexpected exercise type ' + exerciseType);
-        }
+        this.createdPost = this.metisService.createEmptyPostForContext(undefined, undefined, undefined, this.plagiarismCase);
+        // Note the limit of 1.000 characters for the post's content
+        this.createdPost.title = this.translateService.instant('artemisApp.plagiarism.plagiarismCases.notification.title', {
+            exercise: this.exerciseTitle,
+        });
+        this.createdPost.content = this.translateService.instant('artemisApp.plagiarism.plagiarismCases.notification.body', {
+            student: studentName,
+            exercise: this.exerciseTitle,
+            course: courseTitle,
+            cocLink: 'https://www.in.tum.de/fileadmin/w00bws/in/2.Fur_Studierende/Pruefungen_und_Formalitaeten/1.Gute_studentische_Praxis/englisch/leitfaden-en_2016Jun22.pdf',
+            aspoLink: 'https://www.tum.de/studium/im-studium/das-studium-organisieren/satzungen-ordnungen#statute;t:Allgemeine%20Prüfungs-%20und%20Studienordnung;sort:106;page:1',
```

Comment:
Is it really helpful to put these links into the source code and not store them in the json?

Reply:
They do not really fit in a translations file, we have a similar approach in our "About us" page.